source/docs/hardware/hardware-basics/status-lights-ref.rst

```diff
@@ -6,202 +6,511 @@ Many of the components of the FRC Control System have indicator lights that can
++------------------------------+----------------+--------------------------------------------+
+| Both                         | Blinking Red   | Reverse throttle is applied.               |
+|                              |                | Blink rate is proportional to Duty Cycle.  |
++------------------------------+----------------+--------------------------------------------+
+| None                         | None           | No power is being applied to Talon SRX     |
++------------------------------+----------------+--------------------------------------------+
+| LEDs Alternate               | Off/Orange     | CAN bus detected, robot disabled           |
++------------------------------+----------------+--------------------------------------------+
+| LEDs Alternate               | Off/Slow Red   | CAN bus/PWM is not detected                |
++------------------------------+----------------+--------------------------------------------+
+| LEDs Alternate               | Off/Fast Red   | Fault Detected                             |
++------------------------------+----------------+--------------------------------------------+
+| LEDs Alternate               | Red/Orange     | Damanged Hardware                          |
++------------------------------+----------------+--------------------------------------------+
+| LEDs Strobe towards (M-)     | Off/Red        | Forward Limit Switch or Forward Soft Limit |
++------------------------------+----------------+--------------------------------------------+
+| LEDs Strobe towards (M+)     | Off/Red        | Reverse Limit Switch or Reverse Soft Limit |
++------------------------------+----------------+--------------------------------------------+
+| LED1 Only (closest to M+/V+) | Green/Orange   | In Boot-loader                             |
++------------------------------+----------------+--------------------------------------------+
+
++----------------------------------------+
+| B/C CAL Blink Codes                    |
++======================+=================+
+| B/C CAL Button Color | Talon SRX State |
++----------------------+-----------------+
+| Solid Red            | Brake Mode      |
++----------------------+-----------------+
+| Off                  | Coast Mode      |
++----------------------+-----------------+
 
 Spike relay configured as a motor, light, or solenoid switch
 ------------------------------------------------------------
 
 .. image:: images/status-lights/spikeRelay1Light.png
+  :width: 600
+
++---------------------------------+-------------+-----------+-------------------------------------+
+| Inputs                          | Outputs     |           |                                     |
++-----------------+---------------+------+------+ Indicator | Motor Function                      |
+| Forward (White) | Reverse (Red) | M+   | M-   |           |                                     |
```

Comment:
Is there a way to make these 2 header rows stand out?

Reply:
Here's how it looks now:
![Screen Shot 2020-04-30 at 10 15 33 AM](https://user-images.githubusercontent.com/15172054/80739711-bfdac080-8acb-11ea-8f96-e55b7efeedf1.png)
With header rows:
![Screen Shot 2020-04-30 at 10 16 19 AM](https://user-images.githubusercontent.com/15172054/80739727-c6693800-8acb-11ea-826f-3b4ae8e7b3dd.png)

Reply:
ugh... never mind